src/interfaces/uproxy_core_api.ts

```diff
@@ -290,5 +302,12 @@ export interface CoreApi {
   // This may also invoke an OAuth flow, in order to perform operations
   // with the cloud computing provider on the user's behalf.
   cloudInstall(args:CloudInstallArgs): Promise<void>;
+
+  // Invokes OAuth flow and destroys uProxy cloud server on the 
+  // provider specified.
+  cloudDestroy(providerName :string) : Promise<void>;
```

Comment:
Oh, that's cool.

One further suggestion: add an "operation" Enum to CloudOperationArgs with initial values `CREATE` and `DESTROY`. What do you think, would help save on all the boilerplate required to add calls to core?